### ISSUE TITLE
Add thumbnailer

### ISSUE THREAD

**Issue by hadess** [Nokse22/Exhibit#40]
It would be great if a thumbnailer could be added. Of course, it won't work in Flatpaks as thumbnailers are ([rightfully](https://github.com/flatpak/flatpak/issues/4923)) not exported from the sandbox, but this could ([eventually](https://gitlab.gnome.org/GNOME/nautilus/-/issues/946#note_2204397)) be used to create a Flatpak extension to the nautilus Flatpak (or other file manager). Or it could be packaged by distributions in the meanwhile.

**Comment by Nokse22** [comment]
Yes that would be interesting. For the Nautilus extension since it would be separate from the app it could also make sense to just use F3D built in thumbnailer https://f3d.app/doc/user/DESKTOP_INTEGRATION.html#linux

**Comment by hadess** [comment]
Filed https://github.com/f3d-app/f3d/issues/1674 as the thumbnailer there can't be used right now.

**Comment by hadess** [comment]
Maybe Exhibit's thumbnailer could use assimp directly?

I've found this example (which is certainly perfectible):
https://gitlab.com/Kazade/gnome-model-thumbnailer/

**Comment by mwestphal** [comment]
@hadess Exhibit rely on F3D for rendering, so this is not possible.

**Comment by Nokse22** [comment]
It would not be an issue to make a thumbnailer in Exhibit using F3D for rendering, but if it can't be exported with Flatpak it's better to use F3D headless version.
